apps/api-extractor/src/api/model/ApiProperty.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ApiItemKind } from './ApiItem';
+import { ApiDeclarationMixin, IApiDeclarationMixinOptions } from '../mixins/ApiDeclarationMixin';
+import { ApiStaticMixin, IApiStaticMixinOptions } from '../mixins/ApiStaticMixin';
+import { ApiReleaseTagMixin, IApiReleaseTagMixinOptions } from '../mixins/ApiReleaseTagMixin';
+import { ApiResultTypeMixin, IApiResultTypeMixinOptions } from '../mixins/ApiResultTypeMixin';
+import { ApiPropertyItem, IApiPropertyItemOptions } from './ApiPropertyItem';
+
+/** @public */
+export interface IApiPropertyOptions extends
+  IApiDeclarationMixinOptions,
+  IApiReleaseTagMixinOptions,
+  IApiResultTypeMixinOptions,
+  IApiStaticMixinOptions,
+  IApiPropertyItemOptions {
+}
+
+/** @public */
+export class ApiProperty extends ApiDeclarationMixin(ApiReleaseTagMixin(
+  ApiResultTypeMixin(ApiStaticMixin(ApiPropertyItem)))) {
```

Comment:
Can this not fit on one line?

Reply:
Nope

---
In reply to: [236470695](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236470695) [](ancestors = 236470695)